### ISSUE TITLE
ESP32 getting hot but no WIFI Stations showing up

### ISSUE THREAD

**Issue by RapierXbox** [Tnze/esp32_beaconSpam#3]
My ESP32 is just getting hot around the power circutry idk exectly where but a small component near the two leds gets really really hot. i burned myself multible times now. Eaven with a big heastsink getting cooled really good no WIFI Stations show up. Any fixes?

**Comment by RedDoughnut** [comment]
Me too, did you solve the problem?

**Comment by Tnze** [comment]
Is there any Serial output?

**Comment by RapierXbox** [comment]
Yeah its prints the pps, just gets hot. Its not the ESP32 module though its some part of the USB to TTL  _**i think**_ .

**Comment by RapierXbox** [comment]
It should print that "Hello, NodeMCU" must be infront of it

**Comment by RapierXbox** [comment]
Someone else had its esp running hot too and powering it directly with 3.3v fixed it. someone should try as i dont have  a 3.3v supply

**Comment by Tnze** [comment]
You can buy a USB->TTL tool. Most pf them provides 3v3 pin.

**Comment by Behoston** [comment]
For me it works only on very old phone (Samsung A32), but yes, ESP32 is getting hot really fast with this code. I'm using ESP32C3.
